manuscript/060_Statement_First_Programming.md

```diff
@@ -110,13 +110,13 @@ public void ShouldRecognizeTimeSlotAboveMaximumAllowedAsInvalid()
 }
 ```
 
-Note how the method `PerformForTimeSlotIn()`, which triggers the specified behavior is accidentally called BEFORE the mock is actually set up and the set up return value is never taken into account. By some strange coincidence, this error did not alter the expected end result so we did not even notice. It sometimes turns out like this, most often in case of various boundary values (nulls etc.).
+Note how the method `PerformForTimeSlotIn()`, which triggers the specified behavior, is accidentally called *before* the mock is actually set up and the value of `frame.GetTimeSlot_Returns` is not taken into account. Thus this erroneous value does not alter the expected end result, but we'll not notice it. It sometimes turns out like this, most often in case of various boundary values (nulls etc.).
 
 #### 3. Using static data inside production code
 
-Once in a�while, you have to jump in and add some new Statements to some class Specification and some logic to the class itself. Let's assume that the class and its existing specification was written by someone else. Imagine this code is a�wrapper around your product XML configuration file. You decide to write your Statements AFTER applying the changes (�well", you can say, �I am all protected by the Specification that is already in place, so I�can make my change without risking regression, then just test my changes and it is all good...").
+Once in a while, you have to jump in and add some new Statements to an existing class Specification and some logic to the class itself. Let's assume that the class and its specification was written by someone else. Imagine this code is a wrapper around your product XML configuration file. You decide to write your Statements *after* applying the changes (“well", you say, “I am all protected by the Specification that is already in place, so I can make my change without risking regression, and then just test my changes and it is all good...").
```

Comment:
Now that I look at it, "add some new Statements to an existing class Specification and some logic to the class itself" reads weird...

Maybe something like "add some new Statements to an existing Specification and some logic to the class it describes" would be better?